### ISSUE TITLE
appVersion not visible on Bugsnag dashboard with iOS

### ISSUE THREAD

**Issue by henrikra** [bugsnag/bugsnag-react-native#255]
### Description
When creating configuration for bugsnag crash reporter and adding `appVersion` for configuration, it appears that it won't show up with the crash to the dashboard. This is happening only when crash is on iOS
```
const configuration = new Configuration();
configuration.appVersion = 'loool';
```

I can only see these information:
![image](https://user-images.githubusercontent.com/12229968/42319225-016230f2-805a-11e8-91e6-407d52993f2d.png)
Version 9.9.9 is from native.

### Environment

Library versions:

```shell
├─ bugsnag-react-native@2.10.0
├─ react-native-code-push@5.1.1-beta
└─ react-native@0.52.2
```

- iOS/Android version(s): iOS 11.4
- simulator/emulator or physical device?: simulator
- debug mode or production?: both

- [x] (iOS only) `[BugsnagReactNative start]` is present in the
  `application:didFinishLaunchingWithOptions:` method in your `AppDelegate`
  class?

**Comment by henrikra** [comment]
We have `[BugsnagReactNative start] in application:didFinishLaunchingWithOptions` so it can't be that

**Comment by ionut-movila** [comment]
We ran into the same issue. We compute a custom appVersion (different for android and iOS), which is reported correct for android but NOT for iOS. 

Android sample:

`
version 1072
versionCode 2
versionName 1.0.7
`

iOS sample:

`
version 1.0.7
versionCode 2
`

In the sample above iOS should have version 1071(as in our custom computation) and versionName should be 1.0.7 As a result uploaded sourcemaps with 'APP VERSION' 1071 don't match.

**Comment by henrikra** [comment]
@fractalwrench Hmm any news on this? 🤔

**Comment by fractalwrench** [comment]
@henrikra this was an issue with serialisation in the Cocoa library, which has been addressed here: https://github.com/bugsnag/bugsnag-cocoa/pull/292

We're in the process of testing these changes out on React Native, and should hopefully have a release available soon.

**Comment by henrikra** [comment]
Excellent 👍

**Comment by kattrali** [comment]
Fixed in [v2.10.1](https://github.com/bugsnag/bugsnag-react-native/releases/tag/v2.10.1)
